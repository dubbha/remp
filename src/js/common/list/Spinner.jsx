import React from 'react';

const Spinner = () => (
  <div className="list__message">
    <div className="list__spinner" />
  </div>
);

export default Spinner;
