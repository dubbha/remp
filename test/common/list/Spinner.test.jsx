import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Spinner from 'common/list/Spinner';

describe('Spinner', () => {
  it('should render successfully', () => {
    const wrapper = shallow(<Spinner />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
