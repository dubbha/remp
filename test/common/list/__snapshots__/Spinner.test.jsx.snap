// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should render successfully 1`] = `
<div
  className="list__message"
>
  <div
    className="list__spinner"
  />
</div>
`;
